en/views/helpers/form.rst

```diff
@@ -1004,85 +1273,158 @@ Output:
         <option value="value 3" other_attr_name="other_attr_value">Description 3</option>
     </select>
 
-* ``$attributes['multiple']`` If 'multiple' has been set to ``true`` for an
-  input that outputs a select, the select will allow multiple selections::
+**Controlling Select Pickers via Attributes**
 
-    echo $this->Form->select('field', $options, ['multiple' => true]);
+By using specific options in the ``$attributes`` parameter you can control
+certain behaviors of the ``select()`` method.
 
-  Alternatively set 'multiple' to 'checkbox' to output a list of
-  related check boxes::
+* ``'empty'`` - Set the ``'empty'`` key in the ``$attributes`` argument
+  to ``true`` (the default value is ``false``) to add a blank option with an
+  empty value at the top of your dropdown list.
 
-    $options = [
-        'Value 1' => 'Label 1',
-        'Value 2' => 'Label 2'
-    ];
-    echo $this->Form->select('field', $options, [
-        'multiple' => 'checkbox'
-    ]);
+      For example::
 
-  Output:
+          $options = ['M' => 'Male', 'F' => 'Female'];
+          echo $this->Form->select('gender', $options, ['empty' => true]);
 
-  .. code-block:: html
+      Will output:
```

Comment:
all those extra indentation needs to be removed
<img width="882" alt="capture d ecran 2017-01-17 a 13 50 15" src="https://cloud.githubusercontent.com/assets/4977112/22021189/1a2d259e-dcbc-11e6-91a1-26340e19f596.png">